football/README.md

```diff
@@ -3,3 +3,13 @@
 Service that watches current football matches on PA, and when a goal is scored
 uses the Guardian Mobile Notifications service to send a push notification to
 subscribed users.
+
+### Testing in CODE
+
+In order to facilitate testing when there are no matches in progress, you can [force](https://github.com/guardian/mobile-n10n/blob/682289cc9651f4f8475c9efc5eb2a3da35985c15/football/src/main/scala/com/gu/mobile/notifications/football/Lambda.scala#L66-L76) the `CODE` environment to run in the past using [`football-time-machine`](https://github.com/guardian/football-time-machine).
+
+1. Find a match which [recently took place](https://www.theguardian.com/football/results).
+1. Install a debug version of the app and start following a relevant team or match.
+1. Set a date and time for just before the match started (n.b. the [ZonedDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/ZonedDateTime.html) used must be [URL encoded](https://www.urlencoder.org/)) e.g. `curl https://hdjq4n85yi.execute-api.eu-west-1.amazonaws.com/Prod/setDate?startDate=2020-07-26T14%3A59%3A00%2B01%3A00%5BEurope%2FLondon%5D`
+1. Check the [logs](https://logs.gutools.co.uk/s/mobile/goto/cc75dcf9f10d33851bc1bc1e851f96be) to track progress of the match.
+1. Your device should receive notifications when significant match events occur (e.g. kick off, goals, half-time etc.)
```

Comment:
```suggestion
1. Find a match which [recently took place](https://www.theguardian.com/football/results).
2. Install a debug version of the app and start following a relevant team or match.
3. Set a date and time for just before the match started (n.b. the [ZonedDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/ZonedDateTime.html) used must be [URL encoded](https://www.urlencoder.org/)) e.g. `curl https://hdjq4n85yi.execute-api.eu-west-1.amazonaws.com/Prod/setDate?startDate=2020-07-26T14%3A59%3A00%2B01%3A00%5BEurope%2FLondon%5D`
4. Check the [logs](https://logs.gutools.co.uk/s/mobile/goto/cc75dcf9f10d33851bc1bc1e851f96be) to track progress of the match.
5. Your device should receive notifications when significant match events occur (e.g. kick off, goals, half-time etc.)
```

Reply:
If you leave all items in the list as `1.` GitHub adds the correct numbers for you (this is very handy when adding/removing items and reordering steps)!